components/SignupForm.js

```diff
@@ -141,10 +149,11 @@ const SignupForm = () => {
                 })
               }
               placeholder="****"
+              className="bg-transparent"
             />
           </div>
           <div className="w-full mb-4">
-            <button id="signupbutton" className="btn-primary" type="submit">
```

Comment:
It's probably good to keep this type of ID identifiers on CTAs so that it's easy to target them in analytics. Though I haven't tested the latest analytics to see how these things are..